roles/oidcng/tasks/main.yml

```diff
@@ -63,6 +63,13 @@
   notify:
     - "restart oidcng"
 
+- name: copy oidcng well-known discovery configuration
+  template:
+    src: "openid-configuration.json.j2"
+    dest: "{{ oidcng_dir }}/openid-configuration.json"
+  notify:
+    - "restart oidcng"
+
```

Comment:
Can this be a reload?

Reply:
Reload and restart are the same for Spring boot app